app/src/main/java/com/sopt/dive/LoginActivity.kt

```diff
@@ -0,0 +1,176 @@
+package com.sopt.dive
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.rememberLauncherForActivityResult
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.sopt.dive.ui.theme.DiveTheme
+
+class LoginActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+            DiveTheme {
+                LoginScreen()
+            }
+        }
+    }
+}
+
+@Composable
+fun LoginScreen() {
+    val context = LocalContext.current
+
+    var id by remember { mutableStateOf("") }
+    var pw by remember { mutableStateOf("") }
+
+    // 회원가입 결과로 받은 정보 저장
+    var registeredId by remember { mutableStateOf("") }
+    var registeredPw by remember { mutableStateOf("") }
+    var registeredNickname by remember { mutableStateOf("") }
+    var registeredMbti by remember { mutableStateOf("") }
+
+    // registerForActivityResult로 회원가입 결과 받기
+    val signUpLauncher =
+        rememberLauncherForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
+            if (result.resultCode == -1) {
```

Comment:
-1 보단 RESULT_OK라는 상수를 사용해주는게 가독성이 더 좋을 것 같아요

Reply:
저도 동의해요 ~~ !!

Reply:
```suggestion
            if (result.resultCode == RESULT_OK) {
```

이렇게 바꾸었는데 오류가 나나용..?? RESULT_OK 관해서 import가 안되어있는 것 같아요